assets/pages/monthly-contributions/components/paymentMethods.jsx

```diff
@@ -15,22 +16,29 @@ type PropTypes = {
   email: string,
   firstName: string,
   lastName: string,
+  error: ?string,
 };
 
 
 // ----- Component ----- //
 
 function PaymentMethods(props: PropTypes) {
 
-  let content = 'Please fill in all the fields above.';
+  let errorMessage = '';
+  let stripeButton = <StripePopUpButton email={props.email} callback={postCheckout} />;
+  if ((props.firstName === '' || props.lastName === '') && props.error === null) {
+    errorMessage = <ErrorMessage message={'Please fill in all the fields above.'} />;
+    stripeButton = '';
+  }
 
-  if (props.firstName !== '' && props.lastName !== '') {
-    content = <StripePopUpButton email={props.email} callback={postCheckout} />;
+  if (props.error !== null) {
+    errorMessage = <ErrorMessage message={'There was an error processing your payment. Please\u00a0try\u00a0again\u00a0later.'} />;
```

Comment:
The only time we receive a 500 is when there is a communication problem between support-frontend's backend and Amazon state machine. It is not likely that this will happen. Nevertheless, we need to give feedback if this takes place. We can revisit this copy in a future PR.